dev-tools/cmd/buildpgp/build_pgp.go

```diff
@@ -83,7 +81,7 @@ func main() {
 		os.Stdout.Write(data)
 		return
 	} else {
-		ioutil.WriteFile(output, data, 0640)
+		_ = ioutil.WriteFile(output, data, 0640)
```

Comment:
```suggestion
		err := os.WriteFile(output, data, 0640)
		if err != nil {
		  fmt.Fprintf(os.Stderr, "Error writing file: %v\n", err)
		  os.Exit(1)
		}
```
ref. https://pkg.go.dev/io/ioutil#WriteFile